README.md

```diff
@@ -154,26 +156,38 @@ Additionally:
   - The env var `TF_ACC_TEST_PROJECT_SERVICE_ACCOUNT_TOKEN` must be set as a valid token of the service account. Can also be set in the credentials file used by authentication (see [Authentication](#authentication) for more details)
   - The env var `TF_ACC_PROJECT_ID` is ignored
 - For the Load Balancer service:
-  - OpenStack credentials are required, as the acceptance tests use the OpenStack provider to setup the supporting infrastructure
+  - OpenStack credentials are required, as the acceptance tests use the OpenStack provider to set up the supporting infrastructure
     - These can be obtained after creating a user token through the [STACKIT Portal](https://portal.stackit.cloud/), in your project's Infrastructure API page
   - The env var `TF_ACC_OS_USER_DOMAIN_NAME` must be set as the OpenStack user domain name
   - The env var `TF_ACC_OS_USER_NAME` must be set as the OpenStack username
   - The env var `TF_ACC_OS_PASSWORD` must be set as the OpenStack password
 
 **WARNING:** Acceptance tests will create real resources, which may incur in costs.
 
-# Migration
+## Migration
 
 For guidance on how to migrate to using this provider, please see our [Migration Guide](./MIGRATION.md).
 
-# Reporting Issues
+## Reporting Issues
 
-If you encounter any issues or have suggestions for improvements, please open an issue in the repository.
+If you encounter any issues or have suggestions for improvements, please open an issue in the [repository](https://github.com/SchwarzIT/terraform-provider-stackit/issues).

```

Comment:
This is the link to the _other_ provider 👀. This is the correct one:
https://github.com/stackitcloud/terraform-provider-stackit/issues

Reply:
thx a lot :) changed the link